cluster-baseline-settings/aad-pod-identity/aad-pod-identity.yaml

```diff
@@ -223,12 +226,16 @@ spec:
               fieldRef:
                 fieldPath: spec.nodeName
         securityContext:
+          runAsUser: 0
```

Comment:
this value was requested to be higher than 10k before to avoid conflicts with host user tables IIRC.

Reply:
Sorry, not sure what you mean. Before what? It wasn't defined before. This runs as root by design.

Reply:
To expand on the above...

In the prior version of this, `runAsUser` was not needed to be set, because the container itself set the user to root.  Now for added "compliance" -- the container is set to run as a non-root user, in its defination, however, that means this "decision" to run as root is pushed up to `securityContext` -- in practice, it needs to be run as root -- and the new method of this going forward is to designate it as such here, vs in the container.  This was one of the 1.7.0 changes.

I don't know what host user table conflict you're talking about here, unfortunatly.  Since this worked end-to-end, I'm not sure I know how to test for the conflict you mention.

Reply:
@ferantivero -- any chance you'll get a look at this yet today?  I'm thinking of starting on the AKS 1.19 work over this extended weekend, and would love to base it on this if this is good to go.

Reply:
hey there, sorry for the delay in here, for some reason I haven't seen the replies.

> "Before what?"

In the past, we had to run some advisory tooling in our clusters. As a result from it, we were requested to change `runAsUser` values to something higher than `10k`. Otherwise you might run into conflicts with the host's user tables and it is not desired for security reason.

Something I didn't notice in here is that our desired is to run this "as root". If that is the case, please let's discard this comment.